app.rb

```diff
@@ -43,13 +40,14 @@ def hattr(text)
 post '/memos' do
   @title = '新規作成'
   new_memo = Memo.new(nil, params['subject'], params['content'])
-  Memo.save_memos(memos, new_memo)
+  add_memo(new_memo)
+  memos = fetch_db
   redirect '/'
 end
 
 get '/memos/:memo_id' do
   @memo_id = params[:memo_id].to_i
-  memo = memos.find { |memo_block| memo_block.id == @memo_id }
+  memo = memos.find { |memo_block| memo_block.id.to_i == @memo_id }
   @subject = memo.subject
   @content = memo.content
```

Comment:
```suggestion
  @memo = memo
```

項目が2つぐらいならかわいいもんですが、もし20も30もあった場合はデータの移し替えが大変です。
なのでmemoはmemoのままerbに渡しましょう。